account_balance_line/readme/USAGE.rst

```diff
@@ -0,0 +1,8 @@
+#. If you're an accounting adviser, enable developer mode and go to
+   Accounting > Adviser > Journal items.
+#. Or open an account or a partner, and click on Action > Journal items.
+#. You will see a column named "Balance" with the balance of the line.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
Remove this part, as it's now auto-generated

Reply:
Done